File: Assignment16/Question5.c
#include <stdio.h> 

void Pattern(int iRow, int iCol) 
{ 
    
    if (iRow != iCol) 
    { 
        printf("Invalid input.\n"); 
        return; 
    } 

    for (int i = 1; i <= iRow; i++) 
    { 
        
        for (int k = 1; k < i; k++) 
        { 
            printf("\t"); 
        }

        
        for (int j = i; j <= iCol; j++) 
        { 
            printf("%d\t", j); 
        }
        
        printf("\n"); 
    } 
} 

int main() 
{ 
    int iValue1 = 0, iValue2 = 0; 

    printf("Enter number of rows: "); 
    scanf("%d", &iValue1); 

    printf("Enter number of columns: "); 
    scanf("%d", &iValue2); 

    Pattern(iValue1, iValue2); 

    return 0; 
}

File: Assignment50/Question5.java
import java.util.Scanner;

class Pattern 
{
    public void Pattern(int iRow, int iCol) 
    {
        int i = , j = 0 ;

        if (iRow <= 0 || iCol <= 0) 
        {
            return;
        }

        for (i = 1; i <= iRow; i++) 
        {
            for (j = 1; j <= iCol; j++) 
            {
                if (i == 1 || j == 1 || j == iCol || i == iRow || i == j) 
                {
                    System.out.print(j + "\t");
                } 
                else 
                {
                    System.out.print("\t");
                }
            }
            System.out.println();
        }
    }
}

class Question5 
{
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number of rows: ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns: ");
        int iValue2 = sobj.nextInt();

        

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);
    }
}